modules/ztp-image-based-upgrade-installing-lifecycle-agent-using-gitops.adoc

```diff
@@ -0,0 +1,377 @@
++
+--
+.LcaSubscriptionNS.yaml
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-lifecycle-agent
+  annotations:
+    workload.openshift.io/allowed: management
+    ran.openshift.io/ztp-deploy-wave: "2"
+  labels:
+    kubernetes.io/metadata.name: openshift-lifecycle-agent
+----
+
+.LcaSubscriptionOperGroup.yaml
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: lifecycle-agent-operatorgroup
+  namespace: openshift-lifecycle-agent
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "2"
+spec:
+  targetNamespaces:
+    - openshift-lifecycle-agent
+----
+
+.LcaSubscription.yaml
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: Subscription
+metadata:
+  name: lifecycle-agent
+  namespace: openshift-lifecycle-agent
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "2"
+spec:
+  channel: "stable"
+  name: lifecycle-agent
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  installPlanApproval: Manual
+  status:
+    state: AtLatestKnown
+----
+
+.Example directory structure
+[source,terminal]
+----
+├── kustomization.yaml
+├── sno
+│   ├── example-cnf.yaml
+│   ├── common-ranGen.yaml
+│   ├── group-du-sno-ranGen.yaml
+│   ├── group-du-sno-validator-ranGen.yaml
+│   └── ns.yaml
+├── source-crs
+│   ├── custom-crs
+│   │   ├── LcaSubscriptionNS.yaml
+│   │   ├── LcaSubscriptionOperGroup.yaml
+│   │   ├── LcaSubscription.yaml
+----
+--
+
+. Add the CRs to your common `PolicyGenTemplate`.
++
+.Example directory structure
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: "example-common-latest"
+  namespace: "ztp-common"
+spec:
+  bindingRules:
+    common: "true"
+    du-profile: "latest"
+  sourceFiles:
+    - fileName: custom-crs/LcaSubscriptionNS.yaml
+      policyName: "subscriptions-policy"
+    - fileName: custom-crs/LcaSubscriptionOperGroup.yaml
+      policyName: "subscriptions-policy"
+    - fileName: custom-crs/LcaSubscription.yaml
+      policyName: "subscriptions-policy"
+[...]
+----
+
+[id="zp-image-based-upgrade-installing-oadp-with-gitops_{context}"]
+== Installing and configuring the OADP Operator with GitOps ZTP
+
+You can install and configure the OADP Operator with GitOps ZTP well before you want to start the upgrade.
+
+.Prerequisites
+
+* Create a directory called `custom-crs` in the `source-crs` directory. The `source-crs` directory must be located in the same location as `kustomization.yaml` file.
+
+.Procedure
+
+. Create the following CRs in the `openshift-adp` namespace and push them to the `source-crs/custom-crs` directory.
++
+--
+.OadpSubscriptionNS.yaml
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-adp
+  annotations:
+    workload.openshift.io/allowed: management
+    ran.openshift.io/ztp-deploy-wave: "2"
```

Comment:
It would be nice to remove the "workload.openshift.io/allowed: management" annotation since OADP doesn't recognize it
```suggestion
    ran.openshift.io/ztp-deploy-wave: "2"
```